### PR TITLE
JP-2687: Fix a bug in handling single group inputs when abs_refcat is provided

### PR DESCRIPTION
**Pull request by mcara**
Resolves [JP-2687](https://jira.stsci.edu/browse/JP-2687)

Fixes a bug in the logic that handles single image group inputs to tweakreg when `abs_refcat` step parameter was specified.

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [x] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/452/

### CODE REVIEW

**Comment by hbushouse** [comment]
Code updates look OK to me. Do we have a regression test that covers this case?

File: jwst/tweakreg/tweakreg_step.py [hunk 1]
```diff
@@ -252,8 +252,24 @@ def process(self, input):
                 del model.catalog
             return input
 
-        elif len(grp_img) > 1:
+        elif len(grp_img) == 1 and align_to_abs_refcat:
```
[the remark this answers is not in the document]
Comment (mcara) [reply jwst/tweakreg/tweakreg_step.py:255]: If user does not specify `abs_refcat` then only absolute alignment will be turned off. Only the relative alignment will be enabled but it requires at least two images => the message above.

**Comment by mcara** [comment]
@hbushouse I added a unit test for aligning 1 image to an external reference catalog. It is based on existing tests but I had to modify catalogs from `csv` (`cat` extension) to `ecsv` type so that it can work with `abs_refcat`. This is the reason you will see that data files have been modified (no change to the "real" content).

**Review by hbushouse — APPROVED** [review]
LGTM

**Pull request merged**